### ISSUE TITLE
RabbitMQ vars and version control

### ISSUE THREAD

**Issue by jpmjpmjpm** [jcalazan/ansible-django-stack#159]
**RabbitMQ vars overrides the ones defined in env_vars.**

RabbitMQ variables of the playbook:

```
rabbitmq_server_name: "{{ inventory_hostname }}"
rabbitmq_admin_user: admin
rabbitmq_admin_password: password
rabbitmq_application_vhost: "{{ application_name }}"
rabbitmq_application_user: "{{ application_name }}"
rabbitmq_application_password: password
```

are defined in the [env_vars/*.yml](https://github.com/jcalazan/ansible-django-stack/tree/master/env_vars) files as well as in the [roles/rabbitmq/vars/main.yml](https://github.com/jcalazan/ansible-django-stack/blob/master/roles/rabbitmq/vars/main.yml) file.

This is creating issues on my side as I use `ansible-vault encrypt_string` to encrypt rabbitmq password and those passwords are hosts dependent. This seems related to issues #111 and #64 

Suggested fix (before solving issues mentioned in previous paragraph): remove [roles/rabbitmq/vars/main.yml](https://github.com/jcalazan/ansible-django-stack/blob/master/roles/rabbitmq/vars/main.yml) file.

**What do you think?**

Thanks for your feedbacks.

**Comment by YPCrumble** [comment]
@jpmjpmjpm related to this I think our playbook is stuck on version 3.6.15 of RabbitMQ as the debian repo has moved from the apt repository we're referencing. Have you had any issues with your RabbitMQ system? I just had mine stop working after a routine deployment that included some security upgrades on Ubuntu that seems to have broken that version of RabbitMQ.

**Comment by jpmjpmjpm** [comment]
@YPCrumble I haven't notice issues yet with RabbitMQ, but will surely have. I indeed noticed that we're stuck with this old 3.6.15 version. I think that like for PostgreSQL, we should have variables controlling the installed version.

**What do you think?** I can have a look to it.

_As a side comment, I have PR #169 and PR #156 which are still waiting for being merged. The last one since June. Any reason for this???_

**Comment by jpmjpmjpm** [comment]
> @jpmjpmjpm related to this I think our playbook is stuck on version 3.6.15 of RabbitMQ as the debian repo has moved from the apt repository we're referencing. Have you had any issues with your RabbitMQ system? I just had mine stop working after a routine deployment that included some security upgrades on Ubuntu that seems to have broken that version of RabbitMQ.

I create a PR in order to control RabbitMQ installed version. This ensures that a much recent version of RabbitMQ is installed. And also that an adequate version of Erlang is associated.

**Comment by YPCrumble** [comment]
Closed via #174
